client/tests/test_zksnark.py

```diff
@@ -117,19 +121,19 @@
     int("8f167e311eb6fc392551d9d2e87a1c7fc7b022f967f455dd0d60c0dba6943a2d", 16),  # noqa
     int("77c30768bec0349c8351039aef0709c2af413e6ee2dfd13ed418392d06c3f2ff", 16),  # noqa
     # "beta":
-        int("00a172034ccac6782c351ac3a91341b3792d589254c6e8bee9c7becf276bc51f", 16),  # noqa
+    int("00a172034ccac6782c351ac3a91341b3792d589254c6e8bee9c7becf276bc51f", 16),  # noqa
     int("baf9d19645f10d8e4598978a14fb4fb46a7dba2213c47f921d5dfc0318866d5c", 16),  # noqa
     int("e561f69fbabbd1edc3d6fa3f8c4514edb6ab0c93bdd4285e5a25153218fbaa79", 16),  # noqa
-    int("00f96a865caa92048ea60ebd5cf3890b3f550b25597bf304be34ea4dbceb2021", 16),  # noqa
-    int("407975d68a3c0511fa5b039563fbdc34e1a49c4b11766b6c00f4c2592d5c8529", 16),  # noqa
-    int("a56f4d6840730a40e617cd7dfa007df86ffb7dcfea67779d319b798be47c6546", 16),  # noqa
+    int("00297d9e9ed93c0642e1ba82a807763379d10d4528555383944d8ba1c39c9e20", 16),  # noqa
+    int("300230625b48e4070973b769c1b846cfa4e52ca20183922470e81183468f7a04", 16),  # noqa
+    int("f331c95a15f49eb72ff52b30f0f0263f7695c098859888e5c30186741b839b45", 16),  # noqa
```

Comment:
TBH I don't recall exactly.  I possibly dumped the output of the `_to_contract_parameters` function during a successful run of the client tests.  Or maybe manually negated the specific values in python and dumped the hex.

(Note, the actual proofs and keys don't need to be valid since we are only checking the various serializations, so they could be generated from random data)